wsd/DocumentBroker.cpp

```diff
@@ -1787,6 +1795,9 @@ void DocumentBroker::setLoaded()
         _docState.setLive();
         _loadDuration = std::chrono::duration_cast<std::chrono::milliseconds>(
                                 std::chrono::steady_clock::now() - _threadStart);
+        _saveManager.setSavingTimeout(
```

Comment:
Not convinced by this here.

Reply:
It's a way to cap it, with the intent to improve it and tune it. Agreed that it's a bit finger-in-the-air formula, but being able to cap the wait prevents getting stuck. Of course adding tracking of save duration and using that would be better. (We could also make it configurable, but we have too many of those already.)